### DISCUSSION TITLE
0.76.0 for Ubuntu 24.10

### DISCUSSION THREAD

**Discussion by sojusnik** [zim-desktop-wiki/zim-desktop-wiki#2711] [General]
Congrats on the fresh 0.76.0 release!

Would be awesome if you could also release it as a PPA package for Ubuntu 24.10, because at the moment it's only available for [Ubuntu 24.04](https://launchpad.net/~jaap.karssenberg/+archive/ubuntu/zim).

**Comment by jaap-karssenberg** [comment]
Thanks for the heads up. I'm not done yet with publishing the release.
First need to cook new year dinner, then I'll get back to compiling the
windows installer etc.

Op ma 30 dec. 2024 00:19 schreef sojusnik ***@***.***>:
